broker/config/samples/osb_v1alpha1_sfservice.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: osb.servicefabrik.io/v1alpha1
+kind: SFService
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: &id '24731fb8-7b84-4f57-914f-c3d55d793dd4'
+spec:
+  name: &name blueprint
+  id: *id
+  description: &description 'Blueprint service for internal development, testing, and documentation purposes of the Service Fabrik'
+  tags:
+  - 'blueprint'
+  - 'no-not-use'
+  requires: []
+  bindable: true
+  instanceRetrievable: true
+  bindingRetrievable: true
+  metadata:
+    displayName: 'Blueprint'
+    longDescription: *description
+    providerDisplayName: 'SAP SE'
+    documentationUrl: 'https://sap.com/'
+    supportUrl: 'https://sap.com/'
+  dashboardClient:
+    id: blueprint-dashboard-client-id
+    secret: blueprint-dashboard-client-secret
+    redirectURI: 'https://sap.com/'
+  planUpdatable: true
+  context:
+    serviceFabrik:
+      backupEnabled: false
```

Comment:
had this comment in doc review as well. how is backupEnabled not part of metadata and is part of context? same is for image details. lets not add things like service fabrik /operator within service / plan details. Internal impl details should be shown out obtrusively.

Reply:
As discussed already, this is just an example file. I'll remove it.